invenio_administration/templates/semantic-ui/invenio_administration/header/header_login.html

```diff
@@ -24,7 +24,23 @@
         </form>
     {%- else %}
 
+        <div role="menuitem" class="ui dropdown floating rel-p-1 pr-15 computer only">
+            <i class="fitted plus icon inverted"></i>
+            <i class="fitted dropdown icon inverted"></i>
+            <div class="menu">
+              <a class="item black"
```

Comment:
why do we need the `black` class ? the styling is already there since we have the same menu in the frontsite